ISSUE_TEMPLATE/testing.md

```diff
@@ -0,0 +1,99 @@
+First, verify that you meet all the [prerequisites](https://github.com/coreos/fedora-coreos-streams/blob/master/ISSUE_TEMPLATE/release-prereqs.md)
+
+# Pre-release
+
+## Promote testing-devel changes
+
+From the checkout for `fedora-coreos-config` (replace `upstream` below with
+whichever remote name tracks `coreos/`):
+
+- [ ] `git fetch upstream`
+- [ ] `git checkout testing`
+- [ ] `git reset --hard upstream/testing`
+- [ ] `/path/to/fedora-coreos-releng-automation/scripts/promote-config.sh testing-devel`
+- [ ] Sanity check promotion with `git show`
+- [ ] Open PR against the `testing` branch on https://github.com/coreos/fedora-coreos-config
+- [ ] Ideally have at least one other person check it and approve before merging
+
+## Build
+
+- [ ] Start a [pipeline build](https://jenkins-fedora-coreos.apps.ci.centos.org/job/fedora-coreos/job/fedora-coreos-fedora-coreos-pipeline/build?delay=0sec) (select `testing`, and fill in version number using the `30.YYYYMMDD.N` format)
```

Comment:
30 should probably be changed, it wont be 30 forever. should also call out how to determine N.

Reply:
Yes, we should.  Trying to nail it down in https://github.com/coreos/fedora-coreos-tracker/issues/211#issuecomment-518935391.